doc/configuration.md

```diff
@@ -54,6 +54,7 @@ The Ignition configuration is a JSON document conforming to the following specif
   * **_units_** (list of objects): the list of systemd units.
     * **name** (string): the name of the unit. This must be suffixed with a valid unit type (e.g. "thing.service").
     * **_enable_** (boolean): whether or not the service shall be enabled. When true, the service is enabled. In order for this to have any effect, the unit must have an install section.
+    * **_disable_** (boolean): whether or not the service shall be disabled. When true, the service is disabled. In order for this to have any effect, the unit must already be enabled.
```

Comment:
maybe add docs for the behavior if both enable and disable are set, or say the behavior is undefined?